### PR TITLE
New package: iMobieInc.Ashampoo version 26.00.12

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118758)

### CODE REVIEW

**Comment by stephengillie** [comment]
Please add a PackageUrl.

**Comment by upintheairsheep** [comment]
> Please add a PackageUrl.

I did

File: manifests/i/iMobieInc/Ashampoo/26.00.12/iMobieInc.Ashampoo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: iMobieInc.Ashampoo
+PackageVersion: 26.00.12
+Installers:
+- InstallerUrl: https://www.imobie.com/go/download.php?product=ashampoo&id=0
```
Comment (stephengillie) [inline manifests/i/iMobieInc/Ashampoo/26.00.12/iMobieInc.Ashampoo.installer.yaml:7]:
```suggestion
- InstallerUrl: https://cdn1.ashampoo.net/public/imo/6706/ashampoo_winoptimizer_26_38898.exe
```

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

This one got stuck. It should have been approved last week. Sorry about that.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**